coreneuron/utils/randoms/Random123/features/pgccfeatures.h

```diff
@@ -96,7 +96,7 @@ so as not to confuse it with the version available from LANL.
 /* There's no point in trying to compile SSE code in Random123
    unless SSE2 is available. */
 #ifdef __SSE2__
-#define R123_USE_SSE 1
+#define R123_USE_SSE 0
 #else
```

Comment:
Is this necessary change ? could you verify if PGI works with original code?

Reply:
I restored the original behaviour since it seems to be working correctly now.